cpp/dolfin/io/cells.h

```diff
@@ -46,24 +46,26 @@ std::vector<std::uint8_t> lex_to_tp(mesh::CellType type, int num_nodes);
 /// @return The map
 std::vector<std::uint8_t> vtk_to_dolfin(mesh::CellType type, int num_nodes);
 
-/// Convert gmsh cell ordering to FEniCS cell ordering
+/// Convert VTK cell ordering to FEniCS cell ordering
```

Comment:
Where is the FEniCS cell ordering defined? It would be good point to where it's defined.